vendor/assets/manifest.webapp

```diff
@@ -32,6 +32,18 @@
       "folder": "/",
       "index": "index.html",
       "public": false
+    },
+    "/": {
```

Comment:
I agree with @aenario solution: having a `/service-create` route simplify our work later, as we can point to another file (so another app) if we need, or stay pointed to `index.html` to launch our main app and rely on the internal app router to display to right view.

As far as I can remember, we first say that having another app for intents services is a faster way to do it, but maintaining two apps is bit trickier and having a sigle one with a properly configured router is probably the best choice.

So… If we can, I suggest we use the internal router to handle the `/service-create` route in the `index.html` file and trigger the right view/component :smiley: